sdk/confidentialledger/azure-confidentialledger/CHANGELOG.md

```diff
@@ -1,12 +1,23 @@
 # Release History
 
-## 1.0.0b2 (Unreleased)
+## 1.0.0 (Unreleased)
+
+GA Data Plane Python SDK for Confidential Ledger.
 
 ### Bugs Fixed
+- User ids that are certificate fingerprints are no longer URL-encoded in the request URI.
 
-### Other Changes
+### Breaking Changes
+- Removed all models. Methods now return JSON directly.
+- `sub_ledger_id` fields are now named `collection_id`.
+- `azure.confidentialledger.identity_service` has been renamed to `azure.confidentialledger.certificate`.
+- `ConfidentialLedgerIdentityServiceClient` is now `ConfidentialLedgerCertificateClient`.
+- `post_ledger_entry` has been renamed to `create_ledger_entry`.
 
+### Other Changes
 - Python 2.7 is no longer supported. Please use Python version 3.6 or later.
```

Comment:
Do we want to bump this to 3.7 before GA? Or in a subsequent release?

Reply:
We'll bump to 3.7 in anticipation of removing support for 3.6.